AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -45,7 +45,7 @@ windowsobraz.com##.marquee
 ! https://github.com/AdguardTeam/AdguardFilters/issues/34346
 anidub.me,anidub.tv##.foothead
 ! https://github.com/AdguardTeam/AdguardFilters/issues/38034
-dizifilmdergisi.com,kocaelitv.com.tr,turizmhabermerkezi.net,bulten.gen.tr,seskocaeli.com##.okudunuzmu
+bulten.gen.tr,dizifilmdergisi.com,kocaelitv.com.tr,malatyatekhaber.com,mersinistikbal.net,nehir.net,seskocaeli.com,turizmhabermerkezi.net##.okudunuzmu
```

Comment:
don't chande domains order. New domains should be added at the start of the line for element hiding rules, and at the end of domains list for URL rules